### ISSUE TITLE
Baseline+CN

### ISSUE THREAD

**Issue by JingLiJJ** [SankhaSubhra/GAMO#5]
Thanks for your effort. 

Could you please tell me the training strategy of baseline+CN on fashion-mnist? I just thought the baseline+CN follows end-to-end training (backbone network F + classifier M). But, I obtained the accuracy higher than the reuslts on the paper, and it is even very close to the results of GAMO. Look forward to you reply.

**Comment by TanmDL** [comment]
Exactly. With the same architecture as mentioned in the paper,  I have got a better model accuracy of 83.6 for the Fashion-MNIST dataset. Don't even apply feature space augmentation. Simple, backbone network and classifier M with CE loss gives better accuracy than the paper claimed.

**Comment by KAISER1997** [comment]
@TanmDL  I am facing the same issue . Were you able to find any fix to this?

**Comment by Shounak-D** [comment]
@JingLiJJ, @TanmDL, @KAISER1997 There are few reasons I can think of:
1. The shuffling of the dataset and/or classes is different, resulting in a different imbalanced distribution than the one used for the experiments in the paper.
2. The batch sizes are different than used in the paper.
3. This is less likely, but you may be measuring the overall accuracy instead of the ACSA (the mean of the accuracies on the individual classes reported in the paper).

Unless it is the latter point, I suspect that GAMO should still compare favorably to the baselineCN numbers that you get in your setup.
